app-with-router/src/my-app.scss__if_sass

```diff
@@ -0,0 +1,18 @@
+nav {
+	background: #eee;
```

Comment:
Indentation pls. You have `\t` in the file.

Reply:
done